<launch>

    <!--- Run the cpp code -->
    <node pkg="ijcai" type="stage4" name="stage4" output="screen"/>
 
    <!-- Run SR node -->
    <node pkg="xfyun_waterplus" type="iat_node" name="xfyun_iat_node" output="screen">
        <param name="cn" type="bool" value="false"/> 
        <param name="start" type="bool" value="false"/>
        <param name="cue" type="bool" value="true"/>
    </node>

    <!-- Run TTS node -->
    <node pkg="xfyun_waterplus" type="tts_node" name="xfyun_tts_node" />

    <!-- Run sound_play -->
    <node name="soundplay_node" pkg="sound_play" type="soundplay_node.py"/>

</launch>
